internal/testutils/log_pipeline_builder.go

```diff
@@ -33,18 +47,53 @@ func (b *LogPipelineBuilder) WithStatusCondition(cond metav1.Condition) *LogPipe
 	return b
 }
 
+func (b *LogPipelineBuilder) HTTPOutput(opts ...HTTPOutputOption) *LogPipelineBuilder {
+	for _, opt := range opts {
+		opt(b.httpOutput)
+	}
+	return b
+}
+
+func (b *LogPipelineBuilder) LokiOutput(opts ...LokiOutputOption) *LogPipelineBuilder {
+	for _, opt := range opts {
+		opt(b.lokiOutput)
+	}
+	return b
+}
+
+func (b *LogPipelineBuilder) CustomOutput(opt CustomHostOption) *LogPipelineBuilder {
+	b.customOutput = opt()
+	return b
+}
```

Comment:
We don't really need any opts here. You can only provide it as a string
```suggestion
func (b *LogPipelineBuilder) CustomOutput(custom string) *LogPipelineBuilder {
	b.customOutput = custom
	return b
}
```